### PR TITLE
Fix bundler support

### PR DESCRIPTION
**Pull request by ClareCat**
Bundler didn't have the correct rubyopt/includes setup. This adds the info for that so that we don't accidentally load system bundler.

### CODE REVIEW

**Review by mdehoog — APPROVED** [review]

**Comment by heimdall-asguard** [comment]
Review Error for mdehoog @ 2020-06-29 23:31:24 UTC
User must have write permissions to review

File: ruby/private/binary/binary.bzl [hunk 1]
```diff
@@ -18,8 +18,12 @@ def _get_gem_path(incpaths):
     """
     if len(incpaths) == 0:
         return ""
-    incpath = incpaths[0]
-    return incpath.rsplit("/", 3)[0]
+    for incpath in incpaths:
+        if "/lib/ruby/" in incpath and "/gems/" in incpath:
```
Comment (sds) [inline ruby/private/binary/binary.bzl:22]: I'm not sure how careful we need to be, but is it possible for `gems` to be a subdirectory other than in the Ruby install directory? (e.g. if the user's name was "gem" for some reason)

**Review by sds — APPROVED** [review]
Approving with minor comment.

File: ruby/private/binary/binary.bzl (hunk 1 above)
Comment (ClareCat) [reply ruby/private/binary/binary.bzl:22]: No - this isn't the ruby install directory. This is the bundle directory that we create in bazel so its name is not tied to the user's name nor is it tied to the ruby install. The naming will always be very consistent.

Comment (ClareCat) [reply ruby/private/binary/binary.bzl:22]: For example: https://github.com/coinbase/rules_ruby/blob/master/ruby/private/bundle/create_bundle_build_file.rb#L38-L43

We get the incpaths from these generated rules that also must follow this path structure or the code for our gems wouldn't get included in the runfiles directory.

**Pull request merged**